pkg/tidb/client.go

```diff
@@ -33,7 +33,8 @@ var (
 )
 
 const (
-	defaultTiDBStatusAPITimeout = time.Second * 10
+	defaultTiDBStatusAPITimeout      = time.Second * 10
+	defaultTiDBSQLExecutionTimeoutMs = 600000 // 600s
```

Comment:
It is 600s on this line, but the PR title is 60s.

Reply:
Sry, it's a PR title's typo, should be 600s.